examples/02_Parameter Estimation.ipynb

```diff
@@ -187,15 +199,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Sure enough- parameter estimation determined that the thrower's height wasn't 20m, instead was closer to 1.8m, a much more reasonable height!"
+    "Sure enough, parameter estimation determined that the thrower's height wasn't 20m. Instead, it was closer to 1.8m, a much more reasonable height."
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Example 2) Using Tol"
+    "## Using Tol"
```

Comment:
It doesn't allow me to add comments on a line you didn't change, but I suggest changing the next paragraph to

```
The `estimate_params()` `tol` argument can be used to configure the degree to which parameters are optimized. The exact function that the `tol` argument uses is specific to the method used. For example, the `tol` argument for the `Nelder-Mead` method is the change in the lowest error and its corresponding parameter values between iterations. The difference between iterations for both of these must be below `tol` for parameter estimation to converge.
```